src/js/components/Overview/Chart.tsx

```diff
@@ -22,10 +25,23 @@ const Chart = ({ chartType, data, units }: ChartProps) => {
             units={units}
             preFilter={removeMissing}
             dataMap={translateMap}
+            onClick={(d) => {
+              navigate(`${location.pathname.slice(0, 3)}/search?${id}=${d.payload.x}`);
+            }}
           />
         );
       case chartTypes.PIE:
-        return <PieChart data={data} height={CHART_HEIGHT} preFilter={removeMissing} dataMap={translateMap} />;
+        return (
+          <PieChart
+            data={data}
+            height={CHART_HEIGHT}
+            preFilter={removeMissing}
+            dataMap={translateMap}
+            onClick={(d) => {
+              navigate(`${location.pathname.slice(0, 3)}/search?${id}=${d.name}`);
```

Comment:
i think it makes more sense to inject the language directly here rather than slicing it from the location - this is less immediately obvious